### ISSUE TITLE
NTR: 3-hydroxyvaleric acid present in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8592]
New term request 3-hydroxyvaleric acid present in urine (segal_030512172429):

**Comment by pnrobinson** [comment]
@MickeySegal
from this Wikipedia [entry](https://en.wikipedia.org/wiki/3-Hydroxypentanoic_acid) I infer that the "official" name of this compound is [3-hydroxypentanoic acid](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_139272). Is this correct and can we make the primary label "Elevated urine 3-hydroxypentanoic acid level"?

**Comment by MickeySegal** [comment]
Seems right.  All my citations are from Hommes FA (1990) Techniques Diagnostic Human Biochemical Genetics, which seems to use a non-preferred term.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000604
- label: Elevated urinary 3-hydroxypentanoic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: 
- def: The amount of 3-hydroxypentanoic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8592
- synonyms:
